lib/noticed/delivery_methods/one_signal.rb

```diff
@@ -0,0 +1,56 @@
+module Noticed
+  module DeliveryMethods
+    class OneSignal < DeliveryMethod
+      required_options :include_aliases
+
+      def deliver
+        post_request url, headers:, json:
+      rescue Noticed::ResponseUnsuccessful => exception
+        # OneSignal might returns 200 with errors
```

Comment:
![image](https://github.com/user-attachments/assets/e7796127-bff8-4312-a336-e6c341613ef2)

From https://documentation.onesignal.com/reference/push-notification

There's currently no way to handle that because `Noticed::ResponseUnsuccessful` is not raised when the response code is 200.

Reply:
You can parse the response and raise the exception if there is an error in the body.